template-component/src/component/schema.ts

```diff
@@ -2,9 +2,8 @@ import { defineSchema, defineTable } from "convex/server";
 import { v } from "convex/values";
 
 export default defineSchema({
-  counters: defineTable({
-    name: v.string(),
-    value: v.number(),
-    shard: v.number(),
-  }).index("name", ["name", "shard"]),
+  notes: defineTable({
+    text: v.string(),
+    createdAt: v.number(),
```

Comment:
```suggestion
  userId: v.string(), // Note: you can't use v.id referring to external tables
```
we generally prefer to use the built-in _creationTime